code/nathan/python/lab5.py

```diff
@@ -0,0 +1,24 @@
+import random
+
+def pick_6():
+    numbers = []
+    for x in range(6):
+        numbers.append(random.randint(1, 99))
+    return numbers
+
+winning_ticket = pick_6()
+
+count = 0
+
+for x in range(100_000):
+    if winning_ticket == pick_6():
+        count += 1
+
+balance = -2.0
```

Comment:
Right now every response I receive from running the program is 'You won $-2.0! Your ROI is $-2.0' Make sure you test often throughout your code. Consider following each step in order.